### PR TITLE
added basic manual instructions

### PR DESCRIPTION
**Pull request by SwartzCr**
In progress
pulled from joohoi's PR - feel free to comment @joohoi & @bmw

### CODE REVIEW

File: docs/using.rst [hunk 1]
```diff
@@ -55,7 +55,8 @@ standalone_ Y    N    | Uses a "standalone" webserver to obtain a cert. Requires
                       | with no webserver, or when direct integration with the local
                       | webserver is not supported or not desired.
 manual_     Y    N    | Helps you obtain a cert by giving you instructions to perform http-01_ (80) or
-                      | domain validation yourself.                                   dns-01_ (53)
+                      | domain validation yourself. Additionally allows you to        dns-01_ (53)
+                      | specify scripts to be run before and after validation.
```
Comment (pde) [inline docs/using.rst]: Not just "scripts to be run before and after validation", but "scripts to let you automate the validation task in a customized way", or something to that effect?

File: docs/using.rst [hunk 2]
```diff
@@ -168,6 +169,10 @@ the UI, you can use the plugin to obtain a cert by specifying
 to copy and paste commands into another terminal session, which may
 be on a different computer.
 
+Additionally you can specify scripts to run before and after verification
```
Comment (pde) [inline docs/using.rst]: I think people might be confused by this before/after notion. It's true that the scripts run before/after boulder does its validation, but from the user's perspective these scripts actually perform the authentication/validation task, and clean up from it?

Comment (pde) [inline docs/using.rst]: I guess either "perform authentication" or "prepare for validation" could be better ways of putting it?

File: docs/using.rst [hunk 3]
```diff
@@ -416,6 +423,129 @@ The following files are available:
    could convert using ``openssl``. You can automate that with
    ``--renew-hook`` if you're using automatic renewal_.
 
+.. _hooks:
+
+Pre and Post Validation Hooks
+=============================
+
+Certbot allows for the specification fo pre and post validation hooks when run
```
Comment (jsoref) [inline docs/using.rst]: fo => of

**Pull request merged**